tests/mock_helper.py

```diff
@@ -98,7 +98,7 @@ def setup_partitioned_parquet(self):
             for file in files:
                 full_path = os.path.join(subdir, file)
                 keysub = subdir.split(os.path.sep)[
-                    subdir.split(os.path.sep).index('T')+1:]
+                    subdir.split(os.path.sep).index('tmp')+1:]
```

Comment:
Quick pass it looks like the only time this setup function is used right now is in 2 `test_fetch_parq` tests, and we may be explicitly overriding the key in both tests... so dumb luck I guess?

Reply:
Dumb luck indeed, that issue to sunset MockHelper is always at the back of my mind with this stuff